zebra-rpc/src/methods.rs

```diff
@@ -504,30 +505,18 @@ where
         let (tip_height, tip_hash) = self
             .latest_chain_tip
             .best_tip_height_and_hash()
-            .ok_or_else(|| Error {
-                code: ErrorCode::ServerError(0),
-                message: "No Chain tip available yet".to_string(),
-                data: None,
-            })?;
+            .ok_or(to_error("No Chain tip available yet"))?;
```

Comment:
Thank you for the clean up! Could we use the error conversion trait here?

```suggestion
            .ok_or_server_error("No Chain tip available yet")?;
```